test-e2e/cip-auditor/cip-auditor-e2e.go

```diff
@@ -755,26 +820,77 @@ func enablePubSubTokenCreation(
 	return err
 }
 
-func empowerServiceAccount(invokerServiceAccount string) error {
-	args := getCmdEmpowerServiceAccount(invokerServiceAccount)
+func enableCloudResourceManagerAPI(
+	projectID string,
+) error {
+	args := getCmdEnableService(
+		projectID,
+		"cloudresourcemanager.googleapis.com")
+	_, _, err := execCommand("", args...)
+	return err
```

Comment:
This could also be refactored to call a shared method `func enableService(projectID, service string)` ... or a class.